src/integration-tests/tests/auto_populated.rs

```diff
@@ -22,9 +22,9 @@ mod tests {
     mockall::mock! {
         #[derive(Debug)]
         StorageControl {}
-        impl storage_control::stub::StorageControl for StorageControl {
-            async fn create_folder(&self, req: storage_control::model::CreateFolderRequest, _options: gax::options::RequestOptions) -> gax::Result<gax::response::Response<storage_control::model::Folder>>;
-            async fn list_anywhere_caches(&self, req: storage_control::model::ListAnywhereCachesRequest, _options: gax::options::RequestOptions) -> gax::Result<gax::response::Response<storage_control::model::ListAnywhereCachesResponse>>;
+        impl storage::stub::StorageControl for StorageControl {
```

Comment:
This code probably would be more readable with some `use` declarations.

Reply:
Probably, but it is below my line of things that I care about. :grimacing: